ruby/ruby_trunk/ruby_benchmarks/.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: required
+
+services:
+  - docker
+
+before_install:
+  - sudo apt-get install -y curl
+  - sudo apt-get install -y jq
+
+script:
+  - sed -i "s_FROM.*_FROM rubybench/ruby_trunk:$(curl -s -S "https://registry.hub.docker.com/v2/repositories/rubybench/ruby_trunk/tags/" | jq '."results"[]["name"]'| sed -n 1p |cut -c2-8)_" ./Dockerfile
```

Comment:
Ok so our base image is `rubybench/ruby`. Now I  need to keep updating the tag of `rubybench/ruby`. Does the tag for rubybench/ruby_trunk matter to us or we just keep building it from our base image?